superheroes.yaml

```diff
@@ -495,17 +580,20 @@ states:
 
   # Fights
   FIGHTS_ENABLED: true
-  FIGHTS_DB_IMAGE: "mongo:7.0"
+  FIGHTS_DB_IMAGE: "docker.io/library/mongo:7.0"
```

Comment:
Is this image anywhere else? we will be throttled from docker.io, maybe we should publish to quay.io

Reply:
I could not find the corresponding image on quay.io 🫤 

> we will be throttled from docker.io

true, as long as we are not logged in with a docker account

We might have the same problem for `jaeger`, `opentelemtry` and `postgres`

Reply:
I think we can push images to quay.io for our testing

Reply:
As discussed, I update this PR to use the google mirror `mirror.gcr.io` as default registries for some of the images, like `postgres`, `mariadb` and etc.

Reply:
looks good ty